modules/default/weather/forecast.njk

```diff
@@ -20,9 +20,15 @@
                     {{ f.minTemperature | roundValue | unit("temperature") | decimalSymbol }}
                 </td>
                 {% if config.showPrecipitationAmount %}
-                    <td class="align-right bright precipitation">
-                        {{ f.precipitation | unit("precip") }}
-                    </td>
+                    {% if f.precipitationUnits %}	
+                        <td class="align-right bright precipitation">
+                            {{ f.precipitation }}{{ f.precipitationUnits }}
+                        </td>			
+		    {% else %}
```

Comment:
Please check for intendation. Did you also run "npm run lint:prettier" before submitting?

Reply:
Thanks for catching this! I did run prettier but screwed up committing the updates back to my branch. This should be corrected now.